include/dali/core/geom/mat.h

```diff
@@ -539,6 +540,82 @@ std::ostream &operator<<(std::ostream &os, const dali::mat<rows, cols, T> &m) {
   return os;
 }
 
+template <int n, int m, typename T>
```

Comment:
```suggestion
template <int N, int M, typename T>
```
would help understanding that those are template arguments

Reply:
done